extensions/railway/src/railway.ts

```diff
@@ -5,9 +5,9 @@ const backboardUrl = "https://backboard.railway.app/graphql/v2";
 export const railwayWebUrl = "https://railway.app";
 
 export const projectUrl = (projectId: string, page?: string): string =>
-  `${railwayWebUrl}/project/${projectId}/${page ?? "settings"}`;
+  `${railwayWebUrl}/project/${projectId}/${page ?? ""}`;
 
-const token = getPreferenceValues<Preferences>().railwayApiKey;
```

Comment:
Why remove the `Preferences` type? It enforces "token" as string in TS.

Reply:
Sorry, my bad. It was throwing an error and it looked like it worked the same without it, but my TS Server must have been stuck. Changed it back and it works now.

Reply:
Happens to me all the time. I use the command palette in VS Code to `restart TS server` then it usually starts working again.